journal/original/05-chapter-4.adoc

```diff
@@ -280,12 +318,19 @@ Never shall your memory be effaced from my soul, I shall not cease to love you;
 it will be a comfort to my weary soul,
 if sometimes you will make me joyful by imparting a few lines of love,
 and nothing but death will be able to prevent my answering your letters.
-In love I embrace you in my heart, as your ever loving brother,
+In love I embrace you in my heart, as
+
+[.signed-section-closing]
+your ever loving brother,
 
+[.signed-section-signature]
 Marconnay.
 
+[.signed-section-context-close]
 Berlin, 22nd October, 1796.]
 
+--
+
 Returning now to the Journal:
```

Comment:
here too... this would look best with an [.offset]

Reply:
I totally agree. I've been looking back between these corrections of how they were and it's really nice to get a visual of the difference between them.